apiserver/facades/agent/uniter/uniter.go

```diff
@@ -2891,3 +2908,7 @@ func (u *UniterAPI) CloudAPIVersion() (params.StringResult, error) {
 	result.Result = apiVersion
 	return result, err
 }
+
+// RemoveUpgradeCharmProfileData isn't on the v11 API.
+// Note this overwrites the embedded LXDProfileAPI
```

Comment:
Can we remove the LXDProfileAPI for the latest UniterAPI version?

Reply:
This is more tricky that I wanted, but it's achievable

Reply:
I've updated the PR to remove the LXDProfileAPI, hopefully this will be cleaner for future UniterAPIs. https://github.com/juju/juju/pull/10090/commits/4bd8082322967912e6bcac773d98e105e5a3309e